src/Sulu/Bundle/AdminBundle/Admin/View/ViewBuilderTrait.php

```diff
@@ -57,4 +57,12 @@ public function addRerenderAttribute(string $attribute): ViewBuilderInterface
 
         return $this;
     }
+
+    private function addRequestParametersToView(View $route, array $requestParameters): void
```

Comment:
I don't think that this makes sense on the general `ViewBuilderTrait`. I would move it to the traits that actually need this function.

Reply:
cannot do that 🙈 i would have to add it to the `ListViewBuilderTrait` and the `FormViewBuilderTrait` - but then the `FormOverlayListViewBuilder` cannot implement both traits.

Reply:
That's not true - actually we already do that in the [`FormOverlayListViewBuilder` using the `insteadof` keyword](https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Admin/View/FormOverlayListViewBuilder.php#L18-L22), because these two traits already share more method names.

That also brings me to another question: Do we want to support adding different `requestParameters` for the list and form requests?